src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -3,97 +3,77 @@
- // public final JoystickButton intakeGateDownButton = new JoystickButton(SM.xBoxController, RobotMap.intakeGateDownButtonNum);
+  public final CANSparkMax intakePulley = new CANSparkMax(RobotMap.intakeTalonPort, CANSparkMaxLowLevel.MotorType.kBrushed);
+  public final CANSparkMax intakeArm = new CANSparkMax(RobotMap.intakeArmTalonPort, CANSparkMaxLowLevel.MotorType.kBrushed);
+  public final DoubleSolenoid intakeArmSolenoid = new DoubleSolenoid(RobotMap.intakeArmUpNode, RobotMap.intakeArmDownNode);
+  public final DoubleSolenoid gateSolenoid = new DoubleSolenoid(RobotMap.IntakeGateUpNode, RobotMap.IntakeGateDownNode);
+  private boolean isGateOpen = false;
+  //public final DoubleSolenoid humanIntakeSolenoid = SM.getDoubleSolenoid(RobotMap.humanIntakeUpNode, RobotMap.humanIntakeDownNode);
+  //public final iRaidersButton intakeGateUpButton = new iRaidersButton(SM.xBoxController, RobotMap.intakeGateUpButtonNum);
+  //public final JoystickButton intakeGateDownButton = new JoystickButton(SM.xBoxController, RobotMap.intakeGateDownButtonNum);
   //public final JoystickButton intakeOnButton = new JoystickButton(SM.xBoxController, RobotMap.intakeOnButtonNum);
-  public final iRaidersButton intakeReverse = new iRaidersButton(SM.xBoxController, RobotMap.reverseIntakeButtonNum);
-  public final iRaidersButton humanIntake = new iRaidersButton(SM.xBoxController, RobotMap.reverseIntakeButtonNum);
-  private boolean toggleState = true;
-  private boolean testyIntake = false;
-  private double triggerPreviousValue;
+  //public final iRaidersButton intakeReverse = new iRaidersButton(SM.xBoxController, RobotMap.reverseIntakeButtonNum);
+  //public final iRaidersButton humanIntake = new iRaidersButton(SM.xBoxController, RobotMap.reverseIntakeButtonNum);
+  //private boolean toggleState = true;
+  //private boolean testyIntake = false;
+  //private double triggerPreviousValue;
 
 
-  //public final IntakeCountCommand intakeCountCommand = new IntakeCountCommand(this);
-  public final HumanIntakeCommand humanIntakeCommand = new HumanIntakeCommand(this);
-  public final IntakeGateCommand intakeGateCommand = new IntakeGateCommand(this);
-  public final IntakeConveyerCommand intakeConveyerCommand = new IntakeConveyerCommand(this);
+  /*
+    //public final IntakeCountCommand intakeCountCommand = new IntakeCountCommand(this);
+    public final HumanIntakeCommand humanIntakeCommand = new HumanIntakeCommand(this);
+    public final IntakeGateCommand intakeGateCommand = new IntakeGateCommand(this);
+    public final IntakeConveyerCommand intakeConveyerCommand = new IntakeConveyerCommand(this);
+
+   */
   public IntakeSubsystem() {
 
     // Robot.initializeSparkDefaults(intakeMotor, intakeConveyor);
 
   }
 
 
-    public void initTeleop () {
-      //new EncoderReporter(intakeTalon).start();
-
-      //Robot.initializeTalonDefaults(intakeTalon);
-    }
+  public void initTeleop() {
+    //new EncoderReporter(intakeTalon).start();
```

Comment:
on init, we should probably ask the the arm to go up. we can add this in competition though, once we test the merge